packages/graphql-toolbox/src/modules/EditorView/docexplorer/TypeDoc.tsx

```diff
@@ -5,256 +5,236 @@
  *  LICENSE file in the root directory of this source tree.
  */
 
-import React, { ReactNode } from 'react';
+import React, { ReactNode } from "react";
```

Comment:
This file/component is from an external source, almost exclusively prettier updates.